examples/window-app/efr32/src/main.cpp

```diff
@@ -57,10 +57,9 @@ int main(void)
     chip::DeviceLayer::SetDeviceInfoProvider(&gExampleDeviceInfoProvider);
 
     WindowApp & app = WindowApp::Instance();
-
-    EFR32_LOG("Starting App");
```

Comment:
Please revert changes on this files.

Reply:
Called StartApptask function from the main() which will create AppTaskMain task and the Event Queue.
The AppTaskMain will initializing the window application and run the event handler loop.Removed the redundant task exist before and now having only one task and the flow is similar for both WiFi and Thread. 
The design flow is similar to the Light and door-lock apps except the base application class reference.

Reply:
Like I said, in order to preserve the abstraction layer of this app, please remove ALL `AppTask` reference accross ALL files. 

As stated by @rcasallas-silabs most if not all changes required for the support of the WiFi platform should be Added to the init function. There is no need to create another set of init function with a different name for this application.